aggregator/src/aggregator/http_handlers.rs

```diff
@@ -208,6 +208,8 @@ pub(crate) static AGGREGATION_JOB_ROUTE: &str =
 pub(crate) static COLLECTION_JOB_ROUTE: &str = "tasks/:task_id/collection_jobs/:collection_job_id";
 pub(crate) static AGGREGATE_SHARES_ROUTE: &str = "tasks/:task_id/aggregate_shares";
 
+static ROBOTS_TXT: &str = "User-agent: *\nDisallow: /\n";
```

Comment:
I suppose the risk here is that we may disallow some bots that our eventual subscribers _want_ to have trigger report uploads. (concrete example: operational metrics like page load time are plausibly interesting even if the page load is triggered by a bot.) These bots would have to *not* use a deterministic RNG.

The choice we make here will affect all subscribers -- I can't think of a way to make this task-specific (or account-specific, etc).

WDYT about blocking Googlebot specifically, since we know it won't work, but allowing access from others?

Reply:
> I can't think of a way to make this task-specific (or account-specific, etc).

We can change the contents based on the referrer, although this is clumsy at best. I don't think it's worth pursuing.

> WDYT about blocking Googlebot specifically, since we know it won't work, but allowing access from others?

That sounds fine to me.